test/morgan.js

```diff
@@ -822,6 +822,79 @@ describe('morgan()', function () {
         test.write('0')
       })
     })
+
+    describe('a custom', function () {
```

Comment:
Sure, but "a" is an article and "function" in a noun, so it makes sense. I assume that you're not referring to "custom" the noun here, though you're using it as a noun. I guess it sounds weird since the string doesn't end in the noun you are trying to mean.